package.json

```diff
@@ -61,6 +62,7 @@
     "eth-block-tracker": "^8.0.0",
     "execa": "^5.0.0",
     "isomorphic-fetch": "^3.0.0",
+    "jest-silent-reporter": "^0.5.0",
```

Comment:
Given the suggestions above, we may want to move this to the package level.

Reply:
The package-level build scripts seem to be able to find and use `jest-silent-reporter` listed in the root-level package.json. I'm getting the correct output for `"test": "jest"` vs. `"test": "jest --reporters=jest-silent-reporter"`.

Reply:
We could definitely add this to the package level. Would there be any issues with keeping the root-level installation as well? The `test` output doesn't work as expected without the `--reporters=jest-silent-reporter` specified in the root-level script.

Reply:
Hmm, it's just a bit confusing as `jest` isn't defined at all at the top level. I guess if we have to have it here though then that's fine.

Reply:
We could install `jest` at the top level as well. I think we'll need to do that if we want the package-level `test:clean` scripts to run the tests as well. Package size probably shouldn't be affected since `jest` is already used in the subpackages?